admin/linux/debian/debian.xenial/post-patches/qt5.5.patch

```diff
@@ -11,3 +11,14 @@ diff -ruN nextcloud-client-2.4.0.orig/src/CMakeLists.txt nextcloud-client-2.4.0/
  if (Qt5Core_VERSION VERSION_LESS 5.9.0)
  message(STATUS "For HTTP/2 support, compile with Qt 5.9 or higher.")
  endif()
+--- nextcloud-client-2.4.0.orig/admin/osx/CMakeLists.txt	2018-05-22 07:01:48.248646951 +0200
++++ nextcloud-client-2.4.0/admin/osx/CMakeLists.txt	2018-05-22 07:01:51.012689711 +0200
```

Comment:
This doesn't look right

Reply:
If yu mean the ```nextcloud-client-2.4.0``` directory, it does not matter, because patching is done with ```-p1```. (The previous chunk also uses the same directory names).

Reply:
No I mean I don't get why you patch an OSX file ;)

Reply:
I see :) well, I was wondering myself, but  cmake seems to  read and process all CMakeLists.txt files. And the build started to fail when this file was modified to require Qt 5.6.